v2.2/managed-frequently-asked-questions.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Frequently Asked Questions
+summary:
+toc: true
+build_for: [managed]
+---
+
+## Cluster basics
+
+### Is my cluster secure?
+
+Yes. We create individual sub-accounts and VPCs for each cluster within the cloud provider. These VPCs are firewalled from each other and any other outside connection, unless whitelisted for SQL and Web UI ports. The whitelist is an additional layer of protection for your cluster. Connections will only be accepted if they come from a whitelisted IP address, which protects against both compromised passwords and any potential bugs in the server.
```

Comment:
About "Connections will only be accepted..." : It's seems unclear (to me at least, because I don't understand this yet) who's doing the whitelisting - CRL or the customer.